src/services/stellar/contractClient.ts

```
@@ -83,6 +86,32 @@ export class ContractClient {
+  ): Promise<ContractInvokeResult> {
+    return sorobanTracer.startActiveSpan(
+      `soroban.invoke.${options.functionName}`,
+      async (span) => {
+        span.setAttributes({
+          "soroban.contract_id": options.contractId,
+          "soroban.function": options.functionName,
+          "soroban.source_account": options.sourceAccount,
+        });
+        try {
+          const result = await this._invokeContractInner(options);
+          span.setAttributes({ "soroban.tx_hash": result.transactionHash, "soroban.ledger": result.ledger });
```

ESLint

[error] Replace `·"soroban.tx_hash":·result.transactionHash,·"soroban.ledger":·result.ledger` with `⏎············"soroban.tx_hash":·result.transactionHash,⏎············"soroban.ledger":·result.ledger,⏎·········`

(prettier/prettier)